### PR TITLE
2.4 order barplots

### PR DESCRIPTION
**Pull request by martapa**
this is branched from 6.3

Adds functions to order bar plots

### CODE REVIEW

File: src/utils/order.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+export const orderMinority = dataArr => {
+  const descData = dataArr
+    .concat()
+    .sort((a, b) => a.count - b.count)
```
Comment (vindhya) [inline src/utils/order.js]: I think if you do `.sort((a, b) => b.count - a.count` here then you won't have to do the `.reverse()` afterward. Same with the below instances of `.reverse()` as well.

File: src/utils/order.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+export const sortDesc = dataArr => {
+  const descData = dataArr
+    .concat()
+    .sort((a, b) => a.count - b.count)
+    .reverse();
+
+  return descData;
+};
```
Comment (IrinaGM) [inline src/utils/order.js]: Great job on that one! 👍 👍 👍 👍 👍

**Comment by martapa** [comment]
I need to add tests. I thought I can add them in a different PR together with other tests that are missing.

File: src/views/CohortPage.js [hunk 3]
```diff
@@ -39,20 +38,23 @@ const CohortPage = () => {
       </Route>
       <Route path={`${path}/minority-group`}>
         <BarGraph
-          title={`Cohort ${cohortID}: Minority Groups`}
```
Comment (vindhya) [inline src/views/CohortPage.js]: Oh, I just noticed that the Cohort ID is being removed from the graph title due to this change.

Comment (martapa) [reply src/views/CohortPage.js]: I think it might have happened while I was merging master.

File: src/views/CohortPage.js [hunk 4]
```diff
@@ -41,22 +42,22 @@ const CohortPage = ({ theme }) => {
       <Route path={`${path}/minority-group`}>
         <BarGraph
           title={`Cohort ${cohortID}: Minority Groups`}
-          data={cohort.minorityGroup}
-          fillColor={theme.color.aqua}
```
Comment (vindhya) [inline src/views/CohortPage.js]: fillColor is being removed here :)

**Pull request merged**